### PR TITLE
Selenium Smoke Tests

### PR DESCRIPTION
**Pull request by braunsonm**
Runs the Selenium Smoke tests on build to ensure the timeline is rendered properly on our production builds.

### CODE REVIEW

File: scripts/smoke-test.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  def tearDown(self):
+    self.driver.close()
+
+
+if __name__ == "__main__":
+  if len(sys.argv) < 2:
+    print("USAGE: [HTML File]")
+    exit(1)
+
+  SmokeTests.TEST_FILE = sys.argv.pop()
```
Comment (camilleion) [inline scripts/smoke-test.py]: What arguments would you give to represent the test_file?

Comment (braunsonm) [reply scripts/smoke-test.py]: Just the file path ie: `build/index.html`

**Comment by braunsonm** [comment]
Sorry for all the spam, I’m finally done.

**Comment by ets994** [comment]
awesome

**Pull request merged**